lib/pbench/server/report.py

```diff
@@ -180,7 +182,7 @@ def post_status(self, timestamp, doctype, file_to_index=None):
                     fname = "report-status-payload.{}.{}.xz".format(
                         self.name, timestamp_noutc
                     )
-                    fpath = os.path.join(self.config.LOGSDIR, self.name, fname)
+                    fpath = Path(self.config.log_dir) / self.name / fname
```

Comment:
I think I had a comment somewhere earlier that `self.config.log_dir` is a string - if I misled you into  "correcting" this, I apologize, but I now think it *is* a Path (l.140ff in pbench/common/logger.py). But a Path of a  Path is a no-op, so it does not hurt.

Reply:
I had thought you pointed out a real problem.

In all the places where `config.log_dir` is used and assigned, it should be a `string`.  So the code above should be correct: `assert isinstance(self.config.log_dir, string)`.

I'll go review the logger to ensure it is not assigning a `Path` object to `self.config.log_dir`.

Reply:
I am not seeing where we assign a `PbenchConfig` object to have a `log_dir` field with a `Path` value.  Let me know if you find otherwise.

Reply:
Yes, at the place I pointed to, only a local variable `log_dir` is Path-ified. It *was* a real problem :-)